src/lib/components/tx-message/EventBox.tsx

```diff
@@ -48,6 +48,7 @@ export const EventBox = ({ event, msgIndex }: EventBoxProps) => {
             ampCopierSection="tx_page_event_logs"
             fixedHeight={false}
             maxWidth="full"
+            queryParams={{}}
```

Comment:
Since this will be linked to proposal only. Wouldn't it better to not pass the queryParams